src/hooks/useOPL1CrossDomainMessengerProxy.ts

```diff
@@ -41,8 +40,10 @@ async function l2ContractCallInfo({
     address: contract.address,
     abi: contract.abi,
     functionName,
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
     args: args as any, // TODO: TS does not infer correctly the type of valueuseop
     account: walletAddress,
+    // biome-ignore lint/suspicious/noExplicitAny: <explanation>
     value: value as any, // TODO: TS does not infer correctly the type of value
```

Comment:
I think we can move those TODOs to the `<explanation>` part. Aside from the fact that the "TODO" text has nothing to do with a "TODO" sentence.

I'll add it to my ToDo list.

Reply:
Good thing about `TODO` is that it gets highlighted by the IDE. But yeah.